django/db/backends/oracle/schema.py

```diff
@@ -31,10 +31,17 @@ def quote_value(self, value):
         else:
             return str(value)
 
+    def remove_field(self, model, field):
+        # If the column is the identity_column then drop identity is required
```

Comment:
If the column is an identity column, drop the identity before removing the field.